src/benchmarks/beluga_vs_nav2_multi_dataset/reports/nominal_report/sections/datasets/diff_drive_sim_24hs_description.inc

```diff
@@ -0,0 +1,37 @@
+Dataset information
+^^^^^^^^^^^^^^^^^^^
+
+This is a synthetic dataset created recording a simulated robot wandering around a
+450m^2 office environment for 24 hours.
+
+The simulated diff-drive robot is modelled after a Kobuki platform with a mounted RPLidar A1 (range: 8m). The
```

Comment:
@glpuga nit:
```suggestion
The simulated diff-drive robot is modelled after a Kobuki platform featuring an RPLidar A1, an 8m range 2D lidar. The
```
Also, links to Kobuki and RPLidar sites would be nice.

Reply:
Added links.